server/start.go

```diff
@@ -271,6 +273,12 @@ func startStandAlone(svrCtx *Context, svrCfg serverconfig.Config, clientCtx clie
 		return err
 	}
 
+	if opts.PostSetup != nil {
```

Comment:
```suggestion
	if opts.PostSetupStandalone != nil {
```

Reply:
gently ping on this one

Reply:
@itsdevbear, it should still be committed as you've forced pushed your last commit, which removed this.